docs/_posts/2018-03-14-development-update.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Jekyll 4.0 is on the Horizon!"
+date: "2018-03-14 16:07:00 +0100"
+author: oe
+categories: [community]
+---
+
+With the release of Jekyll 3.8.0, it's been 2 and a half years since the last major release. Jekyll 3.0.0 was released in late October of 2015! That's a long time ago, and we've been working towards the next major release of Jekyll for a couple of months now. Here's a small preview of what's to come:
+
+- Dropping support for Ruby 2.1 and 2.2. Both versions are reaching or have reached their EOL period.
```

Comment:
Ruby 2.2 has officially reached EOL.. so the `reaching or..` can be removed..

Reply:
:+1: